src/darwin/Framework/Matter.xcodeproj/project.pbxproj

```diff
@@ -2431,6 +2447,8 @@
 					"MTR_NO_AVAILABILITY=1",
 					"MTR_ENABLE_PROVISIONAL=1",
 					"MTR_ENABLE_UNSTABLE_API=1",
+					"CONFIG_BUILD_FOR_HOST_UNIT_TEST=1",
+					"CHIP_DEVICE_CONFIG_ENABLE_TEST_SETUP_PARAMS=1",
```

Comment:
So this is adding those for the framework build in general, right?  Why do we want that?  I don't think we want CHIP_DEVICE_CONFIG_ENABLE_TEST_SETUP_PARAMS for the framework at all.  And ideally not CONFIG_BUILD_FOR_HOST_UNIT_TEST either.

Reply:
I’ve removed those definitions.